source/includes/fundamentals/auth-specify.rst

```diff
@@ -24,7 +24,7 @@ connection settings when constructing your ``MongoClient``.  Select the
 For more information on these classes and methods, refer to the following API
```

Comment:
Optional suggestion: In the preceding paragraph, I think it's a little confusing to tell users to pick a tab when there aren't any tabs in this section -- maybe you can phrase it to tell users that in each section of the guide, there will be options to select either/or.

Reply:
Updated for clarity